client/components/signInCard.js

```diff
@@ -0,0 +1,116 @@
+"use client";
+
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  CardActions,
+  Divider,
+  FormControl,
+  FormLabel,
+  Link,
+  TextField,
+  Typography,
+} from "@mui/material/";
+import GoogleIcon from "@mui/icons-material/Google";
+
+export default function SignInCard() {
+  async function handleSubmit(e) {
+    //!TODO: eventual code to authenticate login
+  }
+
+  function validateInputs() {
+    //!TODO: code to validate inputs here at some point
+  }
+
+  return (
+    <Card variant="outlined" sx={{ minWidth: 375, p: 2 }}>
+      <CardContent>
+        <Typography
+          component="h1"
+          sx={{
+            width: "100%",
+            fontSize: "clamp(2rem, 10vw, 2.15rem)",
```

Comment:
```suggestion
            fontSize: "2rem",
```
Adding a clamp for the sake of an 8% size increase seems a bit frivolous. I'd go with a simple rem sizing.